netbox/templates/dcim/inc/nonracked_devices.html

```diff
@@ -0,0 +1,62 @@
+{% load helpers %}
+
+<div class="card">
+<h5 class="card-header">
+    Non-Racked Devices
+</h5>
+<div class="card-body">
+{% if nonracked_devices %}
```

Comment:
SiteView needs to be extended to include `nonracked_devices` in its context, otherwise this will never display.